test/zzz_od/operation/choose_predefined_team/test_choose_predefined_team.py

```
@@ -0,0 +1,282 @@
+
+    assert area.text == '预备出战'
+    assert area.lcs_percent == 1.0
+    assert area.color_range == [[240, 240, 240], [255, 255, 255]]
+
+
+def test_choose_team_selects_target_while_confirm_is_disabled(
+        test_context: FixtureTestContext,
+        click_recorder: list[Point],
+        operation: ChoosePredefinedTeam,
+) -> None:
+    """确认按钮为灰色时应选择目标队伍，不得提前进入确认节点。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)